pkg/zonemgr/zone_manager_test.go

```diff
@@ -0,0 +1,52 @@
+package zonemgr
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"os"
+
+	k8stypes "k8s.io/apimachinery/pkg/types"
+)
+
+var _ = Describe("Zone Manager functionality", func() {
+
+	const (
+		customDomain = "domain.com"
+		customNSIP   = "1.2.3.4"
+	)
+
+	var zoneMgr *ZoneManager
+
+	BeforeEach(func() {
+		os.Setenv("DOMAIN", customDomain)
+		os.Setenv("NAME_SERVER_IP", customNSIP)
+		zoneMgr = NewZoneManager()
+	})
+
+	AfterEach(func() {
```

Comment:
There is no need to add an empty `AfterEach`.

Reply:
done